src/monitor.rs

```diff
@@ -113,6 +113,14 @@ pub trait MonitorHandleProvider: AsAny + fmt::Debug + Send + Sync {
     #[cfg_attr(not(web_platform), doc = "detailed monitor permissions.")]
     fn position(&self) -> Option<PhysicalPosition<i32>>;
 
+    /// Returns physical size of monitor in millimeters.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **X11, **Wayland**, **MacOS**, **Windows**: Returns expected values.
+    /// - **Other platforms:** Always returns [`None`].
+    fn physical_size(&self) -> Option<PhysicalSize<u32>>;
```

Comment:
Two notes on the API:
1. `PhysicalSize` might not be the best tool for this? It usually represents pixels, but we have a wildly different measure here.
2. I think we'd want to return `f32`, in case the size has higher precision than 1 millimeter.

Reply:
>I think we'd want to return f32, in case the size has higher precision than 1 millimeter.

it's in millimeter by standard.

Reply:
1. I blindly copied this one. We could return tuple `(u32, u32)` instead. What do you think?
2. Since Apple are the only one providing sub-millimeter distance and I assume that we want to have a uniform cross-platform experience, I think that it would be better to stick with whole numbers. Should I add platform specific functionality for MacOS at `MonitorHandleExtMacOS`?

Reply:
> it's in millimeter by standard.

Which standard?

Reply:
I think he talks about [EDID](https://en.wikipedia.org/wiki/Extended_Display_Identification_Data).

Reply:
yes, EDID.

Reply:
Ah, makes sense. Mind adding a comment about this between the doc comment and `fn`?

And yes, I think I would prefer `(u32, u32)`, `PhysicalSize` (consider e.g. that it'd make no sense to convert it to `LogicalSize`).